_posts/2022-01-17-changelog-163.markdown

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #163"
+date: 2022-01-17 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8018] Add a widget for the [external linter](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-code-analysis.html#call-external-linter).
+Widget indicates whether the linter is turned on. Also, the widget shows a notification when linter significantly affects the code analysis performance.
+The external linter settings were moved to `Preferences | Languages & Frameworks | Rust | External Linters`.
+If necessary, the maximum duration for the external linter can be changed via `Registry... | org.rust.external.linter.max.duration` option
+
+  <img src="/assets/posts/changelog-163/widget_init.png" width="700px"/>
+  <img src="/assets/posts/changelog-163/widget_on.png" width="700px"/>
+  <img src="/assets/posts/changelog-163/widget_disable.png" width="700px"/>
```

Comment:
```suggestion
  <img src="/assets/posts/changelog-163/widget_init.png" width="700px"/>
  
  
  <img src="/assets/posts/changelog-163/widget_on.png" width="700px"/>
  
  
  <img src="/assets/posts/changelog-163/widget_disable.png" width="700px"/>
```

otherwise, it's not clear that there are three separate screenshots